### DISCUSSION TITLE
Cross filtering in Country Map

### DISCUSSION THREAD

**Discussion by Shisir99** [apache/superset#22464] [Q&A / Help]
I have a country chart as shown below of India. I want to apply cross filtering to it, i.e, if i click on a particular state than all other charts show only the values related to that state. Is this possible? 
![Screenshot from 2022-12-23 16-05-58](https://user-images.githubusercontent.com/43596686/209323484-a2ba824d-27d3-4297-b8bb-49f296bc864c.png)

**Comment by yash-hc** [comment]
"Did you get anything?"
